themes/sdss_subtheme/templates/components/news-list/news-item-list.html.twig

```diff
@@ -68,17 +79,26 @@
         </a>
         {%- endif -%}
 
-        {# News List Date #}
-        {%- block block_news_list_date %}
-          {# Since CSS Grid is used, always render an element even if it's empty. #}
-          <div class="su-news-list__item-date">
-            {% if news_list_publishing_date|render|striptags('<drupal-render-placeholder>')|trim is not empty %}
-              <p>{{ news_list_publishing_date|render_clean }}</p>
-            {% endif -%}
-          </div>
-        {% endblock -%}
+        {# News List Source #}
+        <div class="su-news-list__source">
+          {% if news_source|render|striptags('<drupal-render-placeholder>')|trim is not empty %}
+            <p>{{ news_source|render_clean }}</p>
+          {% endif -%}
+        </div>
+
+
+        {# News List External link  #}
+        <div class="news-vertical-teaser__ext-link">{{ su_news_source }}</div>
+
+        {# News List Long Dek #}
+        {# Since CSS Grid is used, always render an element even if it's empty. #}
+        <div class="su-news-list__long-dek">
+          {% if news_list_publishing_date|render|striptags('<drupal-render-placeholder>')|trim is not empty %}
```

Comment:
Should this be checking the value of the long dek field and not the publishing date?

Reply:
The Missing label is a mistake.

Reply:
I fixed this in the new push.